ecl/provider_connectivity/v2/tenant_connections/testing/fixtures.go

```diff
@@ -0,0 +1,639 @@
+        "tenant_id_other": "c7f3a68a73e845d4ba6a42fb80fce03f",
+        "network_id": "061dbaa9-a3e0-4343-b3fc-0a619db66854",
+		"device_interface_id": "interface_2",
+        "device_type": "ECL::VirtualNetworkAppliance::VSRX",
+        "device_id": "c291f4c4-a680-4db0-8b88-7e579f0aaa37",
+        "port_id": "",
+        "status": "active"
+    }
+}
+`
+
+// UpdateRequest provides the input to as Update request.
+const UpdateRequest = `
+{
+  "tenant_connection": {
+    "name": "update_name",
+    "description": "update_desc",
+    "tags": {
+      "update_tags": "update"
+    },
+	"name_other": "update_name_other",
+    "description_other": "update_desc_other",
+    "tags_other": {
+      "test_tags_other": "update"
+    }
+  }
+}
+`
+
+// UpdateResult provides an update result.
+const UpdateResult = `
+{
+    "tenant_connection":{
+        "id": "ea5d975c-bd31-11e7-bcac-0050569c850d",
+        "tenant_connection_request_id": "90381138-b572-11e7-9391-0050569c850d",
+        "name": "update_name",
+        "description": "update_desc",
+        "tags": {
+			"update_tags": "update"
+		},
+        "tenant_id": "c7f3a68a73e845d4ba6a42fb80fce03f",
+        "name_other": "update_name_other",
+        "description_other": "update_desc_other",
+        "tags_other": {
+			"test_tags_other": "update"
+		},
+        "tenant_id_other": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "network_id": "c4d5fc41-b7e8-4f19-96f4-85299e54373c",
+        "device_type": "ECL::Compute::Server",
+        "device_id": "7cc34d4b-a345-4e51-b3d9-62540faca7bf",
+        "device_interface_id": "",
+        "port_id": "c9c3de44-0720-4acd-87c1-9c76f0f77cac",
+        "status": "down"
+    }
+}
+`
+
+// UpdateBlankRequest provides the input to as Update with blank request.
+const UpdateBlankRequest = `
+{
+  "tenant_connection": {
+    "name": "",
+    "description": "",
+    "tags": {
+      "": ""
+    },
+	"name_other": "",
+    "description_other": "",
+    "tags_other": {
+      "": ""
+    }
+  }
+}
+`
+
+// UpdateBlankResult provides an update with blank result.
+const UpdateBlankResult = `
+{
+    "tenant_connection":{
+        "id": "ea5d975c-bd31-11e7-bcac-0050569c850d",
+        "tenant_connection_request_id": "90381138-b572-11e7-9391-0050569c850d",
+        "name": "",
+        "description": "",
+        "tags": {
+			"": ""
+		},
+        "tenant_id": "c7f3a68a73e845d4ba6a42fb80fce03f",
+        "name_other": "",
+        "description_other": "",
+        "tags_other": {
+			"": ""
+		},
+        "tenant_id_other": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "network_id": "c4d5fc41-b7e8-4f19-96f4-85299e54373c",
+        "device_type": "ECL::Compute::Server",
+        "device_id": "7cc34d4b-a345-4e51-b3d9-62540faca7bf",
+        "device_interface_id": "",
+        "port_id": "c9c3de44-0720-4acd-87c1-9c76f0f77cac",
+        "status": "down"
+    }
+}
+`
+
+// UpdateNilRequest provides the input to as Update with nil request.
+const UpdateNilRequest = `
```

Comment:
Did you confirm that this request got accepted?

Reply:
I confirmed that the Update API was accepted with UpdateNilRequest as the request body.
The API log is described below.
```
$ curl -i -1 \
>  -X PUT \
>  -H "X-Auth-Token: ${Token}" \
>  -H "Content-Type: application/json" \
>  -H "Accept: application/json" \
> -d \
> '{
>  "tenant_connection":{
>  }
> }' \
> https://provider-connectivity-lab3ec-ecl.lab.api.ntt.com/v2.0/tenant_connections/0d956a2e-4958-11ea-8088-525400060300
HTTP/1.1 200 OK
Date: Wed, 12 Feb 2020 06:17:43 GMT
Content-Type: application/json
Content-Length: 646
Connection: keep-alive
ECL-Transaction-ID: 4efcf171-e7a5-4242-afec-71c3f600291c
Set-Cookie: TS0183560f=01cce65ce19c56f5dd3d3c990bc5eb1ad95bf02c6e6aa71d3caab32f0e96b7480e76ae3bcb08dfb2f0b6d64adad8bf3f6e26c32987; Path=/

{"tenant_connection": {"id": "0d956a2e-4958-11ea-8088-525400060300", "tenant_id": "fa68e8d56ff7468d8fb1b056bf13bf48", "tenant_id_other": "d2f19c353e6d4c519e530c6a78438b33", "tenant_connection_request_id": "147c4ffa-481e-11ea-8088-525400060300", "network_id": "15e79044-c6f7-4f92-981d-6ed52b9b5a54", "device_type": "ECL::Baremetal::Server", "device_id": "0acab22f-8993-451c-8a6b-398b0244f578", "device_interface_id": "46eb7624-d462-46c2-8ac7-f988a15d3280", "port_id": "87449d66-4e99-4cf7-9b93-9f153548ccc7", "status": "active", "name": "name", "description": "abc", "tags": {"a": "1"}, "name_other": "", "description_other": "", "tags_other": {}}}
```